api.py

```diff
@@ -332,8 +327,13 @@ def post(self, user_id):
                     INSERT INTO users_convos (convo_id, user_id)
                         VALUES (%s, %s)
                     '''
-            recipients_list = recipients.split(',')
```

Comment:
I think you deleted this line by accident. We don't have recipient_list anywhere else in the code. Re add this line to make the list so we can loop through.